helpdesk_mgmt/controllers/main.py

```diff
@@ -12,8 +14,64 @@ def support_ticket_close(self, **kw):
         values = {}
         for field_name, field_value in kw.items():
             values[field_name] = field_value
-        ticket = http.request.env['helpdesk.ticket'].sudo().\
-            search([('id', '=', values['ticket_id'])])
-        ticket.stage_id = values.get('stage_id')
+        ticket = http.request.env['helpdesk.ticket'].search([(
+            'id', '=', values['ticket_id'])])
+        stage_id = int(values.get('stage_id'))
+        ticket.sudo().update(
+            {'stage_id': stage_id})
 
         return werkzeug.utils.redirect("/my/ticket/" + str(ticket.id))
+
+    @http.route('/new/ticket', type="http", auth="user", website=True)
+    def create_new_ticket(self, **kw):
+        categories = http.request.env['helpdesk.ticket.category']. \
+            search([('active', '=', True)])
+        email = http.request.env.user.email
+        name = http.request.env.user.name
+
```

Comment:
Why leaving this meaningless empty lines inside the method? PEP8 discourages to do that.